configure.py

```diff
@@ -583,7 +588,9 @@ def has_re2c():
              'hash_collision_bench',
              'manifest_parser_perftest',
              'clparser_perftest']:
-  objs = cxx(name)
+  if platform.is_windows():
```

Comment:
This should also be:
```suggestion
  if platform.is_msvc():
```